.circleci/config.yml

```diff
@@ -1175,6 +1184,20 @@ jobs:
     steps:
       - soltest_all
 
+  t_ubu_soltest_deprecated_evm_versions: &t_ubu_soltest_deprecated_evm_versions
+    <<: *base_ubuntu2204_large
+    parallelism: 50
```

Comment:
But how to check the CircleCI credits per commit? Their `insights` feature seems to show only credits in the last 24 hours minimum

Reply:
I was thinking more about just roughly looking at the total time as a proxy for this (basically, the "Timing" tab). I don't know how to check credits directly either, which is actually annoying.

Reply:
Right. In terms of total time, running with more parallelism in a large machines does not seems to help.

Running it with a `medium` machine and parallelism `10`, the slowest run took around [5m30s](https://app.circleci.com/pipelines/github/ethereum/solidity/32748/workflows/e90c6168-905e-4963-8da9-5dfedcb503bd/jobs/1467997/timing) while running with a `large` machine and parallelism `50` it took around [5m52s](https://app.circleci.com/pipelines/github/ethereum/solidity/32749/workflows/3e670703-207d-47a9-9941-c8b22fd4c7bc/jobs/1468075/timing).

Reply:
But the 5:30 and 5:52 are the times of the longest run. I meant the *total time*, because this is how CircleCI charges us (it does not charge for the "idle time").

Looks like this metric is not displayed in their UI but I quickly summed it up and the it actually turns out in favor of `large` with 50 parallel runs (43 min). On `medium` the longest run finishes faster, but it takes 59 min in total. I mean, not sure if the machine size or parallelism has a bigger influence here, but let's just go with the original settings.

I updated the PR to switch it back to `large` and 50 parallel runs.